src/index.html

```diff
@@ -3,11 +3,379 @@
+                <picture>
+                  <source media="(min-width: 745px)" srcset="images/bike-battery-desktop.png">
+                  <source media="(min-width: 668px)" srcset="images/bike-battery-tablet.png">
+                  <img
+                    src="images/bike-battery-mobile.png"
+                    alt="Battery attached to the bike"
+                    class="product-details__image"
+                  >
+                </picture>
+
+                <picture>
+                  <source media="(min-width: 745px)" srcset="images/bike-light-reflector-desktop.png">
+                  <source media="(min-width: 668px)" srcset="images/bike-light-reflector-tablet.png">
+                  <img
+                    src="images/bike-light-reflector.png"
+                    alt="bike light eluminator"
+                    class="product-details__image"
+                  >
+                </picture>
+              </div>
+
+              <h3 class="product-details__article-title">Range & Integrated lights</h3>
+
+              <p class="product-details__paragraph">
+                The removable battery has up to 70km battery autonomy
+                and weighs only 2.4 kg. Lights integrated into the
+                frame give you always-on visibility day and night.
+              </p>
+            </article>
+
+            <article class="product-details__article">
+              <div class="product-details-images-wrapper is-active">
+                <picture>
+                  <source media="(min-width: 745px)" srcset="images/bike-brake-desktop.png">
+                  <source media="(min-width: 668px)" srcset="images/bike-brake-tablet.png">
+                  <img
+                    src="images/bike-brake-mobile.png"
+                    alt="bike brakes handle"
+                    class="product-details__image"
+                  >
+                </picture>
+
+                <picture>
+                  <source media="(min-width: 745px)" srcset="images/bike-stand-desktop.png">
+                  <source media="(min-width: 668px)" srcset="images/bike-stand-tablet.png">
+                  <img
+                    src="images/bike-stand-mobile.png"
+                    alt="bike stand"
+                    class="product-details__image"
+                  >
+                </picture>
+              </div>
+
+              <h3 class="product-details__article-title">
+                Hydraulic disc brakes & Lightweight
+              </h3>
+
+              <p class="product-details__paragraph">
+                Brakes with total stopping power the second you make
+                contact. The removable battery has up to 70km battery
+                autonomy and weighs only 2.4 kg. Lights integrated
+                into the frame give you always-on visibility day and night.
+              </p>
+            </article>
+          </div>
+
+          <button class="product-details__button">Explore</button>
```

Comment:
i guess it should be a link to a home page - enables to check how smooth scroll works